openedx/features/badging/models.py

```diff
@@ -15,26 +16,9 @@
 log = logging.getLogger('edx.badging')
 
 
-class Badge(models.Model):
-    BADGE_TYPES = (
-        badge_constants.CONVERSATIONALIST,
-        badge_constants.TEAM_PLAYER
-    )
+class BadgeManager(models.Manager):
+    """Custom manager for retrieving badges"""
 
-    name = models.CharField(max_length=255, blank=False, null=False)
-    description = models.TextField(blank=True, null=True)
-    threshold = models.IntegerField(blank=False, null=False)
-    type = models.CharField(max_length=100, blank=False, null=False, choices=BADGE_TYPES)
-    image = models.CharField(max_length=255, blank=False, null=False)
-    date_created = models.DateTimeField(auto_now=True)
-
-    class Meta:
-        app_label = 'badging'
-
-    def __unicode__(self):
-        return self.name
-
-    @classmethod
     def get_badges_json(cls, badge_type):
```

Comment:
Since `get_badges_json` is not class method now, first argument should be named `self`

Reply:
Good point 👍 
I renamed it, You can verify it on this PR 
https://github.com/philanthropy-u/edx-platform/pull/1377